### PR TITLE
fix: ito show default contract

### LINKED ISSUE #3039: [Bug] It filled the default qualification address if back to the ITO composition dialog
### How to reproduce?

1. Check the Plugin Contract checkbox w/o input any value in the Plugin Contract field.
2. Click the Next button and jump to the confirmation page.
3. Click the Back button and back to the composition page.
4. It will fill the Plugin Contract field with the default qualification address.

image
image

### Mask Version:

- [x] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [x] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by zhouhanseng**
closes #3039

### CODE REVIEW

**Review by guanbinrui — DISMISSED** [review]

File: packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx [hunk 1]
```diff
@@ -171,7 +171,11 @@ export function CreateForm(props: CreateFormProps) {
     }, [])
 
     // qualificationAddress
-    const [qualificationAddress, setQualificationAddress] = useState(origin?.qualificationAddress || '')
+    const [qualificationAddress, setQualificationAddress] = useState(
+        origin?.qualificationAddress && origin?.qualificationAddress !== DEFAULT_QUALIFICATION_ADDRESS
+            ? origin?.qualificationAddress
```
Comment (UncleBill) [inline packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx:176]: nit:

```suggestion
        origin?.qualificationAddress && origin.qualificationAddress !== DEFAULT_QUALIFICATION_ADDRESS
            ? origin.qualificationAddress
```

**Review by UncleBill — COMMENTED** [review]
No need nullish operators in following expressions

**Review by guanbinrui — APPROVED** [review]

**Review by UncleBill — APPROVED** [review]

File: packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx [hunk 2]
```diff
@@ -171,7 +171,11 @@ export function CreateForm(props: CreateFormProps) {
     }, [])
 
     // qualificationAddress
-    const [qualificationAddress, setQualificationAddress] = useState(origin?.qualificationAddress || '')
+    const [qualificationAddress, setQualificationAddress] = useState(
+        origin?.qualificationAddress && origin.qualificationAddress !== DEFAULT_QUALIFICATION_ADDRESS
```
Comment (septs) [inline packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx:175]:
```suggestion
        origin.qualificationAddress !== DEFAULT_QUALIFICATION_ADDRESS
```

Comment (zhouhanseng) [reply packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx:175]: No, your syntax results in undefined.

**Review by septs — APPROVED** [review]

**Pull request merged**